### DISCUSSION TITLE
How to override schedule time?

### DISCUSSION THREAD

**Discussion by stefan-g** [renovatebot/renovate#29129] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Gitlab

### Please tell us more about your question or problem

I haven't found any documentation about the feature: https://github.com/renovatebot/renovate/pull/2145
I'm using a selfhosted gitlab renovate runner and i'm looking for a way to override the schedule plan of the projects to force  immediate pr creation / merge branches. any hints how to do it? could think about something like to define temporary a env var when the renovate runner scheduler is triggered manually...

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
Try using checkedBranches

**Reply by stefan-g** [reply]
doesn't seems to be the feature i'm looking for (https://docs.renovatebot.com/self-hosted-configuration/#checkedbranches). it only rebase branches but not close them. and all the branches has to be listed. i'm more looking for a way to force the renovate-runner to ignore the configred project schedule.

**Reply by rarkins** [reply]
Do you mean ignore all schedules altogether?

In that case you can configure an open schedule within a "force" object in your global config:

```
{
  "force": {
    "schedule": "at any time"
  }
}
```

**Reply by stefan-g** [reply, marked as answer]
yes, i would like to ignore all schedules. I will try that. thanks

**Reply by stefan-g** [reply]
Thanks, that worked for me. Defined a temporary env  var RENOVATE_CONFIG={"force": {"schedule": "at any time"}}. It will override all project specific schedule times.
